css-layout-classes/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>Choose your character</h1>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="Link" />
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="Yoshi" />
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="column-one-fourth">
```

Comment:
We actually don't need additional `container`s after the first one. The container is setting our max-width height, and spacing, but all we really need are the `row` flex elements which are giving us our `display: flex`. 

While it may not be _necessary_, it's not a bad idea to have a `section` here for the visual break which may have been your plan here too.